CMakeLists.txt

```diff
@@ -133,6 +133,9 @@ message(STATUS "QUDA target: ${QUDA_TARGET_TYPE}")
 # ######################################################################################################################
 
 # build options
+option(QUDA_LARGE_KERNEL_ARG "enable large kernel arg support" OFF)
```

Comment:
just wondering if this should move to the target cuda specific part and if we should check for CUDA >= 12.1 there as well

Reply:
e312d10888a28f9c1ab9f63a2f73ff63e0f9e2c5